### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by HellHunterMax**
Merging dev with main so we have the most up to date version of information for the project in main.
mostly for Contribution, README and rules of conduct.

### CODE REVIEW

**Review by code-reaper08 — APPROVED** [review]
Checked CodeofConduct and Contributing docs.

**Review by chrisK00 — APPROVED** [review]

File: Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs [hunk 1]
```diff
@@ -2,16 +2,24 @@
...
 {
     public static class ServiceExtensions
     {
-        public static IServiceCollection ConfigureAppServices(this IServiceCollection services)
+        public static IServiceCollection ConfigureAppServices(this IServiceCollection services, IConfiguration config)
         {
+            services.AddDbContext<DataContext>(options =>
+                options.UseSqlite(config.GetConnectionString("DefaultConnection")));
+
+
```
Comment (JBSE) [inline Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs:19]: I'm being Pedantic! But we should look at implementing style cop or setting up an editor config to maintain coding standard.

**Review by JBSE — APPROVED** [review]

**Comment by JBSE** [comment]
Not sure if my comment on the review is saved after I approved it. 

I think we should add style cop or include an editorconfig to enforce coding standards.

**Review by N1c0z — APPROVED** [review]

File: Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs (hunk 1 above)
Comment (HellHunterMax) [reply Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs:19]: Yes that would be good! you could open an issue if you like.
If you know how you can also pick it up ?

**Pull request merged**